searchcore/src/vespa/searchcore/proton/metrics/executor_metrics.cpp

```diff
@@ -7,16 +7,19 @@ namespace proton {
 void
 ExecutorMetrics::update(const vespalib::ThreadStackExecutorBase::Stats &stats)
 {
-    maxPending.set(stats.maxPendingTasks);
+    maxPending.set(stats.queueSize.max());
     accepted.inc(stats.acceptedTasks);
     rejected.inc(stats.rejectedTasks);
+    const vespalib::ThreadStackExecutorBase::Stats::QueueSizeT & qSize = stats.queueSize;
```

Comment:
Consider using auto.

Reply:
Fixed in https://github.com/vespa-engine/vespa/pull/12711/commits/a4f448633278360f7cb8ef2135f68291cb021a9f